mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -196,6 +201,8 @@ private void processRecords(List<Record> parsedRecords, JobExecution jobExecutio
       }
       case DELETE_RECORD -> deleteRecords(parsedRecords, jobExecution, params)
         .onSuccess(ar -> promise.complete(parsedRecords)).onFailure(promise::fail);
+      case CREATE_ORDER -> sendEvents(parsedRecords, jobExecution, params, DI_MARC_BIB_FOR_ORDER_CREATED)
```

Comment:
should we consider renaming this event since we no longer create SRS MARC Bib for Orders?

Reply:
how about `DI_INCOMING_MARC_BIB_ORDER_PARSED`?

Reply:
DI_INCOMING_MARC_BIB_FOR_ORDER_PARSED